server/vitals.js

```diff
@@ -121,6 +121,26 @@ async function checkHeartbeat() {
   }
 }
 
+// Check for internal issues by calling DB, if there is, then log error in Sentry
+async function checkForInternalProblems() {
+  const firmwareStateMachineLocations = await db.getActiveFirmwareStateMachineLocations()
```

Comment:
I think I might have removed this function with the latest commit with the idea that `getLocations` could be used instead. Would that still work for you?

Reply:
Thank you for pointing this out! I changed my code to use `getLocations` instead and I think it still works!